pybricks/common/pb_type_speaker.c

```diff
@@ -22,34 +23,90 @@
 #include <pybricks/util_mp/pb_obj_helper.h>
 #include <pybricks/util_pb/pb_error.h>
 
+#define MP_PI MICROPY_FLOAT_CONST(3.14159265358979323846)
```

Comment:
We should change the `MP_` prefix here. Unless we want to suggest making this macro public in upstream MicroPython.

Reply:
OK, will rename it to `PI` only. Pity such a constant is not defined somewhere already.

Reply:
Actually, will delete it, since the we use square wave only at the moment.